lang/system/ja.xml

```diff
@@ -97,6 +102,7 @@
 		<file_corrupted>ファイルが壊れています。</file_corrupted>
 		<microphone_disabled>マイクを有効にしてください。</microphone_disabled>
 		<no>いいえ</no>
+		<ok>OK</ok>
```

Comment:
no need add it, the same name exists in english

Reply:
Yes, is Ok for all

Reply:
whichever, but done.